content/en/blog/_posts/2018-05-29-announcing-kustomize.md

```diff
@@ -8,12 +8,11 @@ date: 2018-05-29
 
 [**kustomize**]: https://github.com/kubernetes-sigs/kustomize
 [hello world]: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/helloWorld
-[kustomization]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#kustomization
 [mailing list]: https://groups.google.com/forum/#!forum/kustomize
 [open an issue]: https://github.com/kubernetes-sigs/kustomize/issues/new
-[subproject]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/0008-kustomize.md
+[subproject]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/2377-Kustomize/README.md
 [SIG-CLI]: https://github.com/kubernetes/community/tree/master/sig-cli
-[workflow]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/workflows.md
+[workflow]: https://github.com/kubernetes-sigs/kustomize/blob/1dd448e65c81aab9d09308b695691175ca6459cd/docs/workflows.md
```

Comment:
I couldn't find where the page moved to, so I made it check the commit from 4 years ago.